test/e2e/tests/account/forgot-password.spec.ts

```diff
@@ -10,20 +9,10 @@ import HomePage from '../../page-objects/pages/home/homepage';
 import LoginPage from '../../page-objects/pages/login-page';
 import ResetPasswordPage from '../../page-objects/pages/reset-password-page';
 import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import { seeAuthenticationRequest } from './mocks/authentication';
 
 const newPassword = 'this is the best password ever';
 
-async function seeAuthenticationRequest(mockServer: Mockttp) {
-  return await mockServer
-    .forPost('https://authentication.api.cx.metamask.io/api/v2/srp/login')
-    // the goal is to know when this request happens, not to mock any specific response
-    .thenCallback(() => {
-      return {
-        statusCode: 200,
-      };
-    });
-}
-
```

Comment:
:information_source:  this has been moved in a re-usable function, together with the lineage mocks